packages/opencode/src/cli/cmd/auth.ts

```diff
@@ -10,6 +10,39 @@ import { Global } from "../../global"
 import { Plugin } from "../../plugin"
 import { Instance } from "../../project/instance"
 
+// Simple command parser that handles quoted strings
+function parseCommand(input: string): string[] {
```

Comment:
This function is definitely a bit "sloppy", I'm open to other approaches. Initially I just wanted to enable the command format in config. Adding the prompts to `auth login` required parsing user input though (that would invariably include some type of quote).